Doc/library/tkinter.messagebox.rst

```diff
@@ -0,0 +1,39 @@
+:mod:`tkinter.messagebox` --- Tkinter message prompts
+=====================================================
+
+.. module:: tkinter.messagebox
+   :platform: Tk
+   :synopsis: Various types of alert dialogs
+
+**Source code:** :source:`Lib/tkinter/messagebox.py`
+
+--------------
+
+The :mod:`tkinter.messagebox` module provides a template base class as well as
+a variety of convenience methods for commonly used configurations. The message
+boxes are modal and will return a subset of (True, False, OK, None) based on
```

Comment:
"The message or alert boxes are modal.  When the user dismisses the box, the functions return one of 
{'ok', 'cancel', 'yes', 'no', 'retry', 'abort', 'ignore'}, as specified below.

Reply:
I ran each of the dialog boxes when I initially wrote this. 'abort', 'retry', 'cancel' and 'ignore' are only returned internally. To the user, the values returned are as I described (ie. return statement in askretrycancel is as follows: 
https://github.com/python/cpython/blob/2af9f5d334eeca588eeee87d177faf0012c63a03/Lib/tkinter/messagebox.py#L119

Please advise